src/main.py

```diff
@@ -72,6 +79,7 @@ def create_account():
         return Response("{'error': 'Error adding the item'}", mimetype='application/json', status=500)
     return Response(json.dumps(add_items), mimetype='application/json', status = 201)
 
+
 @app.route('/Account/Get/<int:acc_num>', methods = ['GET'])
 def get_account_details(acc_num):
     items = actions.get_account_details(acc_num)
```

Comment:
Comment applies for all the print statements in the file.

print statement at line 86 can be ignored.
![image](https://user-images.githubusercontent.com/70574489/217150722-1a591bbf-f26e-4087-b96d-fd4d3bbd57ad.png)